fast/stages/01-resman/organization.tf

```diff
@@ -58,7 +63,10 @@ module "organization" {
       "roles/compute.xpnAdmin" = [
         module.branch-network-sa.iam_email
       ]
-      "roles/orgpolicy.policyAdmin" = local.branch_teams_pf_sa_iam_emails
+      "roles/orgpolicy.policyAdmin" = concat(
```

Comment:
You actually don't need to be policy admin for the data platform, you can set policies at the project level (owner) or folder level (folder admin). I would just remove this.